src/duplo_resource/cronjob.py

```diff
@@ -11,9 +12,14 @@ def __init__(self, duplo: DuploClient):
   
   @Command()
   def list(self):
-    """Retrieve a list of all services in a tenant."""
+    """Retrieve a list of all cronjob in a tenant."""
     tenant_id = self.tenant["TenantId"]
-    return self.duplo.get(f"v3/subscriptions/{tenant_id}/k8s/cronJob")
+    tenant_name = self.tenant["AccountName"]
+    response = self.duplo.get(f"v3/subscriptions/{tenant_id}/k8s/cronJob")
+    if not response.json():
```

Comment:
This can use a walrus operator to avoid running the json() method twice. Like this. 
```
if (data := response.json):
  return data
else:
  raise DuploError(f"No CronJobs found in tenant '{tenant_name}'", 404)
```